### ISSUE TITLE
[File] Accessibility

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1051]
### Issue and Steps to Reproduce

The input file accessibility is not great. The input is in display: none, which is not a good practice.

#### Versions

2.0.0

#### Expected

Change the input design so that it is not visible but still focusable for users.

**Comment by MartinWeb** [comment]
Report sur le nouveau design system
